src/components/molecules/pager/pager.scss

```diff
@@ -0,0 +1,76 @@
+.pager__navigation,
+.pager__container {
+	align-items: center;
+	display: flex;
+	justify-content: flex-start;
+	margin-left: 0;
+}
+
+.pager__container {
+	.pager__navigation {
+		color: #25619b;
+		list-style-type: none;
+
+		padding: 0;
+		margin-bottom: 0;
+
+		& > li {
+			align-items: center;
+			cursor: pointer;
+			display: inline-flex;
+			justify-content: center;
+			list-style-type: none;
+			margin: 0 2px 0;
+		}
+
+		.pager__button {
+			border: none;
+			border-radius: 3px;
+			display: block;
+			min-width: 24px;
+			padding: 2px 6px;
+			background-color: #fff;
+			color: #2b7bba;
+			font-size: 100%;
+			font-family: Noto Sans, Century Gothic, Arial, sans-serif;
+			cursor: pointer;
+			text-decoration: none;
+			white-space: nowrap;
+
+			&.active {
+				background: #25619b;
+				color: #fff;
+				&:hover {
+					background: #25619b;
+				}
+			}
+
+			&.hidden {
+				display: none;
+			}
+
+			&:hover {
+				background: #cbe8fc;
+			}
+		}
+
+		.pager__ellipsis--right,
+		.pager__ellipsis--left {
+			padding: 2px 4px;
+			cursor: default;
+		}
+	}
+}
+
+/*@include break(medium) {
+	.pager__navigation,
+	.pager__container {
+		margin-left: auto;
+	}
+
```

Comment:
If this not needed should probably be removed

Reply:
Updated